gulpfile.js

```diff
@@ -8,13 +8,16 @@ var repoRoot = __dirname + '/';
 var npmRoot = repoRoot + 'node_modules';
 var govukToolkitRoot = npmRoot + '/govuk_frontend_toolkit';
 var govukToolkitAppFolder = repoRoot + 'pages_builder/govuk_frontend_toolkit';
+var govukElementsRoot = npmRoot + '/govuk-elements-sass';
+var govukElementsRootFolder = npmRoot = 'pages_builder/govuk-elements-sass'
```

Comment:
This seems weird to me. Why is `npmRoo`t being reassigned to the same thing as `govukElementsRootFolder`?